packages/mobile/src/components/button/button.tsx

```diff
@@ -1,17 +1,80 @@
 import React, { FC } from 'react';
-import { Pressable } from 'react-native';
+import { Pressable, View } from 'react-native';
+import { ButtonType } from '~/common/enums/enums';
 import { Text } from '~/components/components';
 import { styles } from './styles';
 
+//mock before update global theme color
+enum AppColor {
+  YELLOW_100 = '#F1B313',
+  YELLOW_200 = '#FFB800',
+  GRAY_200 = '#DFDFDF',
+  WHITE = '#FFFFFF',
+}
+////
+
 type Props = {
   label: string;
```

Comment:
```suggestion
  label: ReactText;
```

Reply:
ReactText from React is deprecated. I'll add this type in the component